cmd/amp/cli/lookup/regex-lookup.yml

```diff
@@ -1,8 +1,9 @@
 logs-all: ([a-zA-Z0-9\.\-:\"\s\_\t\n$&?/\[\]%><,;()+='*{}\\]*)
-logs-metadata: (timestamp:"[A-Z0-9\.\-:\"]+\s+time_id:"[A-Z0-9\.\-:\"]+\s+service_id:"[a-z0-9$\"]+\s+service_name:"[a-z0-9\-$\"]+\s+message:"[a-zA-Z0-9\./\-:\s\_\"=\t\n\$\\&%?\[\]\%\>\<\,\;\+$\"]+\s+container_id:"[a-z0-9$\"]+\s+node_id:"[a-z0-9$\"]+\s+task_id:"[a-z0-9$\"]+\s+task_name:"[a-z0-9\.\-$\"]+)
+logs-metadata: (timestamp:"[A-Z0-9\.\-:]+"\s+time_id:"[A-Z0-9\.\-:+]+"\s+service_id:"[a-f0-9]+"\s+service_name:"[a-zA-Z0-9\-]+"\s+message:"[a-zA-Z0-9\./\-:\s\_\"=\t\n\$\\&%?\[\]\%\>\<\,\;\+$]+"\s+container_id:"[a-f0-9]+"\s+node_id:"[a-f0-9]+"\s+stack_id:"[a-f0-9]+"\s+stack_name:"[^"]+")
```

Comment:
@generalhenry can you please show your output, because it was not working for me before the change, the fields were wrong (task_id instead of stack_id, etc) and the regexp didn't have all the characters (+ in the time_id, etc).